PG.StarWarsGame.Files.MEG/PG.StarWarsGame.Files.MEG/Binary/File/Builder/V1/MegFileBuilder.cs

```diff
@@ -42,6 +46,61 @@ public MegFile FromBytes(byte[] byteStream)
             return new MegFile(header, megFileNameTable, megFileContentTable);
         }
 
+        public MegFile FromHolder(MegFileHolder holder, out List<string> filesToStream)
+        {
+            List<string> files = holder.Content.Select(megFileDataEntry => megFileDataEntry.RelativeFilePath).ToList();
+            List<MegFileNameTableRecord> megFileNameTableRecords =
+                files.Select(file => new MegFileNameTableRecord(file)).ToList();
+            megFileNameTableRecords.Sort();
```

Comment:
Why do we need to sort them? I'd image this is user code.

Reply:
Mostly to stay consistent with the base game's archives which are sorted by the CRC checksum:
> [The File Table is sorted on the CRC (in ascending order).](https://modtools.petrolution.net/docs/MegFileFormat)

I would assume they are sorted due to Checksum-based lookups.